src/main/java/com/danielflower/apprunner/App.java

```diff
@@ -116,8 +116,8 @@ public void start() throws Exception {
         String viaName = config.get("apprunner.proxy.via.name", "apprunner");
 
 
-        int idleTimeout = config.getInt("apprunner.proxy.idle.timeout", 30000);
-        int totalTimeout = config.getInt("apprunner.proxy.total.timeout", 60000);
+        int idleTimeout = config.getInt("apprunner.proxy.idle.timeout", 10 * 60 * 1000);
+        int totalTimeout = config.getInt("apprunner.proxy.total.timeout", 10 * 60 * 1000);
```

Comment:
I think these timeouts apply to requests/responses against an already running app. They were quite aggressive to prevent misbehaving apps using up too many resources. I'd rather not change the defaults without thinking about it a bit more, but you can change the values in your own config file.